recipes/skyfield/meta.yaml

```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "skyfield" %}
+{% set version = "0.9" %}
+{% set sha256 = "0f944793cfe63c897252c589ce29e4cc71dbafce1056d657517c02bfeb8e8a72" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - py2compat.patch
+
+build:
+  skip: true  # [win64]
```

Comment:
Maybe not but how we are going to test without this test dependency?
The problem is the failed test is really minor it is a least significant digit +/-1 difference. IMHO it is not a failure at all. I can block the test sgp4 and make it deploy on all platforms but the question is should we do something like that? In my opinion in this case it is justified in this case (the error is really a standard case for the floating point computations).